### ISSUE TITLE
Errors when attempt to search or create a note

### ISSUE THREAD

**Issue by donlovett** [dendronhq/dendron#647]
something went wrong. please submit a bug report to https://github.com/dendronhq/dendron/issues/new?assignees=&labels=&template=bug_report.md&title= with the output of `Dendron: Open Log`

I could not get the output of the log
Attempting to uninstall and reinstall the extension
Renaming the Dendron Folder
Reinitializing the repository

**Comment by donlovett** [comment]
Seemed to recreate and is now working with rename of directory and  initialize workspace command

**Comment by kpathakota** [comment]
Hmm... interesting. It's hard to follow without logs/repeatable repro steps. Is everything working as expected now? I'm closing this issue but if you're able to repro this again/get logs we can take a look.

**Comment by donlovett** [comment]
Hi Kiran

 

Thanks for the follow-up

 

Yes it is working

 

Thanks

Don
